### ISSUE TITLE
Seemingly random repos

### ISSUE THREAD

**Issue by ChrisChinchilla** [MeetHyde/MeetHyde#5]
Hey all,

Super keen to get started with Hyde and help work on the project too.

Having an odd problem getting started though… I've created and added an access token and am seeing a random bunch of repositories, some of which are mine, some of which I've never seen before and none of my jekyll repos.

Any ideas or more information you need?

**Comment by MeetHyde** [comment]
Hi Chris that is super weird, I just test it out with multiple accounts and it's showing the necessary repos, could you please post an image of what you're getting in the home page and tell me which version are you using?

**Comment by ChrisChinchilla** [comment]
Using the latest version, screenshot here. As you can see this is a tiny selection of my repos plus a few I have no idea about.
![screenshot 2015-11-29 09 31 15](https://cloud.githubusercontent.com/assets/42080/11456072/03b9213c-967c-11e5-836b-ae4eda7591a4.png)

**Comment by MeetHyde** [comment]
I see, so the expected is that it shows only the first 5 repos. I settled up a trial for you to use the public API call instead of the Authorized API call. Please open the file again in the browser and let me know.

Beside the first 5 repos appearing in the image you posted, which other repos do you expect to see listed? It seems only the first 5 contains a "gh-pages" branch which are the repos tracked by the app.

**Comment by ChrisChinchilla** [comment]
Hmm, so default behaviour is to only show five repos not all? And only gh-pages branches? I don't use GitHub pages at all and I think earlier Hyde versions allowed for this, so maybe it's time I started a fork to 'fix' this?

**Comment by MeetHyde** [comment]
Yes it's intended to work with GitHub pages so since the beginning it only tracks gh-pages branches, although you can fork the repo and in the file "js/js.js" find the line # 193 which holds this condition and unwrap it's contents, it should now show all your repos.

Please note if you fork the repo it's possible you see the unexpected behavior you commented before since what I did before was just a trial, I'm still thinking what can be causing that and you can take a look at the function showing the repos: Repos() to see if you can spot that before I do.
